{
  "IssueID": 7413,
  "VulnerabilityID": "CVE-2017-7478",
  "Release": "3.5",
  "Package": "openvpn",
  "Repository": "main",
  "FixedVersion": "2.3.15-r0",
  "Subject": "[3.5] openvpn: Multiple vulnerabilities (CVE-2017-7478, CVE-2017-7479)",
  "Description": "\r\n*CVE-2017-7478*: OpenVPN version 2.3.12 and newer is vulnerable to unauthenticated Denial of Service of server via received large control packet. \r\n\r\nh3. Fixed In Version:\t\r\n\r\n*openvpn 2.3.15*, openvpn 2.4.2\r\n\r\nReferences:\r\n\r\nhttps://community.openvpn.net/openvpn/wiki/QuarkslabAndCryptographyEngineerAudits\r\nhttp://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-7478\r\n\r\nPatch:\r\n\r\nhttps://github.com/OpenVPN/openvpn/commit/feb35ee5cac605edddd6e9dc62941e2c53f96fb3\r\n\r\n\r\n\r\n\r\n*CVE-2017-7479*: OpenVPN versions before 2.3.15 and before 2.4.2 are vulnerable to reachable assertion when packet-ID \r\ncounter rolls over resulting into Denial of Service of server by authenticated attacker.\r\n\r\nh3. Fixed In Version:\t\r\n\r\n\r\n*openvpn 2.3.15*, openvpn 2.4.2\r\n\r\n\r\nh3. References:\r\n\r\nhttps://community.openvpn.net/openvpn/wiki/QuarkslabAndCryptographyEngineerAudits\r\nhttps://nvd.nist.gov/vuln/detail/CVE-2017-7479\r\n\r\nh3. Patch:\r\n\r\nhttps://github.com/OpenVPN/openvpn/commit/b727643cdf4e078f132a90e1c474a879a5760578\r\n\r\n\r\n\r\n\r\n\r\n"
}